### ISSUE TITLE
DISK NON-RAID flag generates a warning

### ISSUE THREAD

**Issue by cslashnet** [dangmocrang/check_idrac#59]
Hi,

I just configured your plugin with nagios and I managed to make it work with a server that uses iDrac9, I had to use the latest version you posted on another issue because I was having the same problem with warnings on disks when queried singularly.
But I still have warnings on most of my disks like this one:

PDisk 5 (0:1:0) 3726.02 GB: NON-RAID(!), PowerStat: SPUNUP, HotSpare: no [TOSHIBA, HDD, S/N: 39A8K5Z7F7DE] isFailing: 0

This will generate a warning on nagios I am guessing because of the NON-RAID flag. The thing is that it is supposed to be like that because those disks are not attached to a raid controller as it is an array of disks used in a ZFS pool.

I think it would be needed a flag to suppress this warning.

Otherwise great plugin!

Thanks.

**Comment by eeshlomi** [comment]
Hi there,
Such a flag is already supported! Just add the following parameter:
--ok="ok|online|on|spunup|full|ready|enabled|presence|non-raid"
eeshlomi

**Comment by cslashnet** [comment]
Hi! thanks for the answer! But now I have a warning on all the disks, even those that were ok:

WARN - PDisk 10 (0:1:5) 3726.02 GB: NON-RAID, PowerStat: SPUNUP, HotSpare: no [TOSHIBA, HDD, S/N: 39A8K5YVF7DE] isFailing: 0(!)

**Comment by eeshlomi** [comment]
So let's add 0 as well:
--ok="ok|online|on|spunup|full|ready|enabled|presence|non-raid|0"

**Comment by cslashnet** [comment]
Ok, I see, it is working now thanks!

**Comment by matejzero** [comment]
Just some extra info. I just got a new R6151 server and it reports the following values:
```
IDRAC-MIB-SMIv2::physicalDiskSmartAlertIndication.1 0
IDRAC-MIB-SMIv2::physicalDiskPowerState.1 on
```

So I had to add `0` and `on` to OK states. Do you think this would be good to add to sample config file?
